public/app/features/alerting/unified/components/settings/ExternalAlertmanagers.tsx

```diff
@@ -0,0 +1,126 @@
+import { capitalize } from 'lodash';
+import React from 'react';
+
+import { Badge, Button, Card, Stack, Text, TextLink } from '@grafana/ui';
+import { DATASOURCES_ROUTES } from 'app/features/datasources/constants';
+import { AlertmanagerChoice } from 'app/plugins/datasource/alertmanager/types';
+
+import { ExternalAlertmanagerDataSourceWithStatus } from '../../hooks/useExternalAmSelector';
+import {
+  isAlertmanagerDataSourceInterestedInAlerts,
+  isVanillaPrometheusAlertManagerDataSource,
+} from '../../utils/datasource';
+import { createUrl } from '../../utils/url';
+import { ProvisioningBadge } from '../Provisioning';
+import { WithReturnButton } from '../WithReturnButton';
+
+import { useSettings } from './SettingsContext';
+
+interface Props {
+  onEditConfiguration: (dataSourceName: string) => void;
+}
+
+export const ExternalAlertmanagers = ({ onEditConfiguration }: Props) => {
+  const {
+    externalAlertmanagerDataSourcesWithStatus: externalAlertmanagersWithStatus,
+    deliverySettings,
+    enableAlertmanager,
+    disableAlertmanager,
+  } = useSettings();
+
+  // determine if the alertmanger is receiving alerts
+  // this is true if Grafana is configured to send to either "both" or "external" and the Alertmanager datasource _wants_ to receive alerts.
+  const isReceivingOnAlertmanager = (
+    externalDataSourceAlertmanager: ExternalAlertmanagerDataSourceWithStatus
+  ): boolean => {
+    const sendingToExternal = [AlertmanagerChoice.All, AlertmanagerChoice.External].some(
+      (choice) => deliverySettings?.alertmanagersChoice === choice
+    );
+    const wantsAlertsReceived = isAlertmanagerDataSourceInterestedInAlerts(
+      externalDataSourceAlertmanager.dataSourceSettings
+    );
+
+    return sendingToExternal && wantsAlertsReceived;
+  };
+
+  return (
+    <Stack direction="column" gap={0}>
+      {externalAlertmanagersWithStatus.map((alertmanager) => {
+        const { uid, name, jsonData, url } = alertmanager.dataSourceSettings;
```

Comment:
I would be tempted to have the Alertmanager card as a separate component, and this file then just renders an array of them, an then we can reuse it for the internal one as well - or at least some reusable component so the core parts of both are kept consistent with each other more easily. Can always be done afterwards though

Reply:
I've been putting that off for too long, now's as good a time as any :D I'll include the changes in this PR